docker-files/zeeguu-api-core/Dockerfile

```diff
@@ -24,17 +25,39 @@ RUN python setup.py install
 # https://github.com/zeeguu-ecosystem/Python-Wordstats/blob/master/wordstats/config.py#L1
 # under /opt/Zeeguu-API/wordinfo.db
 COPY --chown=www-data:www-data Zeeguu-API /opt/Zeeguu-API
+RUN rm -rf /opt/Zeeguu-API/.git
```

Comment:
This is probably not needed anymore since we are not going to rely on the .git folder to fetch the versions and we get it from the config file instead.